src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsCallout/ConditionSummary.tsx

```diff
@@ -159,11 +160,13 @@ const ConditionSummary: React.FC<IConditionSummaryProps> = (props) => {
 
     return (
         <Stack tokens={STACK_TOKENS}>
-            <h4 className={classNames.title}>
-                {conditionType === 'boolean'
-                    ? t(LOC_KEYS.flyoutValueTitle)
-                    : t(LOC_KEYS.flyoutValuesTitle)}
-            </h4>
+            {!isNumericType(conditionType) && (
```

Comment:
from an accessibility perspective should this be a header? I feel like a callout should have a "header" to denote the content inside. If we are changing this for styling reasons, we should just override the defaults on the headers.

Reply:
Actually I changed it because I believe it should not be a header. It is a label for either string or boolean input, don't think it needs to be an h4.